changelog.d/pa-123.added

```diff
@@ -0,0 +1 @@
+`.h` files will now run when C or C++ are selected as the language.
```

Comment:
we should grab a more recent semgrep_interfaces that modify generate.py instead of modifying the autogenerated lang.json.

Reply:
Was not aware of that, I'll fix it